sale_margin_delivered_security/models/sale_order_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _name = "sale.order.line"
+    _inherit = ["sale.order.line", "product.cost.security.mixin"]
```

Comment:
Yo don't really need to mix it in again 
```suggestion
    _inherit = "sale.order.line"
```